assets/src/edit-story/components/library/panes/media/common/mediaElement.js

```diff
@@ -160,11 +160,7 @@ function Element({
         if (mediaElement.current && hoverTimer == null) {
           const timer = setTimeout(() => {
             if (activeRef.current && src) {
-              const playPromise = mediaElement.current.play();
-              if (playPromise) {
-                // All supported browsers return promise but unit test runner does not.
-                playPromise.catch(() => {});
-              }
```

Comment:
Can we leave the comment here?

Reply:
It‘s not correct anymore because it‘s now always a promise